docs/licensing.md

```diff
@@ -4,8 +4,9 @@ hide:
 ---
 
 Both [Tart Virtualization](https://github.com/cirruslabs/tart) and [Orchard Orchestration](https://github.com/cirruslabs/orchard)
-are licensed under [Fair Source 100 License](https://fair.io/). Usage on personal computers including personal workstations is royalty-free,
-but organizations that exceed a certain number of server installations utilizing 100 CPU cores will be required to obtain a paid sponsorship.
+are licensed under [Fair Source License](https://fair.io/). Usage on personal computers including personal workstations is royalty-free,
+but organizations that exceed a certain number of server installations (100 CPU cores for Tart and/or 4 hosts for Orchard)
```

Comment:
LGTM overall, but are you sure that we need to mention Orchard here?

Orchard [has it's own license](https://github.com/cirruslabs/orchard/blob/main/LICENSE), and we can change it the same way separately, also this is the only mention of Orchard in this legal text, and to some it might not be evident what we're talking about here.

Reply:
Yeah, but the sponsorship agreement covers both licenses. I think it's fair to have it here until Orchard is that big we have a separate site.